datafusion/physical-plan/src/joins/sort_merge_join.rs

```diff
@@ -3643,102 +3664,12 @@ mod tests {
 
     #[tokio::test]
     async fn test_left_semi_join_filtered_mask() -> Result<()> {
-        let schema = Arc::new(Schema::new(vec![
```

Comment:
why are these tests removed?

Reply:
they are not removed, the common test data preparation was factored out in helper method